examples/rhino-sdk/aggregate_quantile_example.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6eb8e5ca",
+   "metadata": {},
+   "source": [
+    "# Example: Federated Percentiles with Differential Privacy\n",
+    "\n",
+    "### Load all necessary libraries, including 'rhino_health'"
```

Comment:
IMO it's common and clear what the first block of imports does, so we can avoid a bit of extra text here.

```suggestion
```

Reply:
The only reason I kept this was consistency with the other notebooks in tutorials, etc.